src/effects/native/echoeffect.cpp

```diff
@@ -26,26 +27,28 @@ EffectManifest EchoEffect::getManifest() {
 
     EffectManifestParameter* delay = manifest.addParameter();
     delay->setId("delay_time");
-    delay->setName(QObject::tr("Delay"));
```

Comment:
Delay is IMHO the correct name for this parameter and should be kept. Time is only the physical quantity, without further meaning.